stable/artifactory-ha/Chart.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: v1
 name: artifactory-ha
 home: https://www.jfrog.com/artifactory/
+<<<<<<< HEAD
```

Comment:
@danielezer Looks like merge conflict.

Reply:
Whoops, sorry I didn't commit my conflict resolutions. Sorry about that